EpicLoot/src/Magic/MagicItemEffects/ModifySummonHealth.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections.Generic;
+using HarmonyLib;
+using UnityEngine;
+
+namespace EpicLoot.MagicItemEffects;
+
+public class ModifySummonHealth
```

Comment:
Similar to ModifyWispRange, reduce if checking nesting here by returning.

Reply:
Fixed in: https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/024acaca83da30b8d3086d4368ee787e151c8b60